src/index.js

```diff
@@ -0,0 +1,25 @@
+import './index.css';
+import { addMyNewScore, getAllMyGameScores } from './modules/score.js';
+import { newScoreForm, newRefBtn } from './modules/varibles.js';
+
+newScoreForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const myUserInput = newScoreForm.elements.user;
+  const myScoreInput = newScoreForm.elements.score;
+
+  const newScore = {
+    user: myUserInput.value,
+    score: Number(myScoreInput.value),
+  };
+
+  addMyNewScore(newScore);
+
+  myUserInput.value = '';
+  myScoreInput.value = '';
+});
```

Comment:
- [ ] The job of this submit function is to post the form data to the API endpoint only but after I click on the submit button I can see the leaderboard list updated if even I did not click on the `refresh button` which should not be that. I have to get the newly added data if only I click on the `refresh button` is clicked.

Reply:
@Owusu-Desmond,
thanks for your good and valuable feedback.
I will take in to account your nice suggestion in my next attempt.